src/masonite/dot/dot.py

```diff
@@ -0,0 +1,169 @@
+import pydoc
+import inspect
+from backpack import Collection as collect
+
+
+class Dot:
```

Comment:
I guess we already have this implementation in masonite.utils.structures 🤔 ?

Reply:
I think you're right lol